services/feedz/feedz.tester.js

```diff
@@ -46,6 +48,14 @@ t.create('multi-page')
     color: 'blue',
   })
 
+t.create('multi-page-no-items')
+  .get('/feedz/v/shieldstests/public/Shields.MultiPageNoItems.json')
+  .expectBadge({
+    label: 'feedz',
+    message: 'v1.0.0',
+    color: 'blue',
+  })
```

Comment:
I gather from the target that this has been setup specifically for our needs to reproduce the scenario, and we don't have to worry about this changing upstream/invalidating the test, correct?

Reply:
Correct, I've prepared these repos just for this integration. The code is public, although I haven't written the scripts necessary to upload the packages (but I can write them down if there is a need). I can also transfer the repo to you/shields org (although that _might_ pose some problems, as Feedz lacks in some areas).

Reply:
No worries, what you've done is fine; just wanted to confirm!